### ISSUE TITLE
MaxLineLength should count characters instead of bytes

### ISSUE THREAD

**Issue by Claymore** [jorisroovers/gitlint#16]
I would love to use gitlint with commits in Russian but it counts line sizes in bytes instead of characters. Other programs count characters (vim-git and Gitlab, to name a few).

**Comment by jorisroovers** [comment]
Thanks. I'll look into this for the upcoming 0.8.0 release which I'm hoping to spend some time on in the next month or 2. 

If you could give me an example commit message that I can use to test this out with, that would definitely be helpful :-)

**Comment by Claymore** [comment]
Thank you in advance! The example:
```
root@ubuntu1404amd64:~# echo "RM #15259 [HD @52271] Проставляем временные метки" | gitlint --ignore B6
1: T1 Title exceeds max length (74>72): "RM #15259 [HD @52271] Проставляем временные метки"
root@ubuntu1404amd64:~# echo "RM #15259 [HD @52271] Проставляем временные метки" | wc -c
75
root@ubuntu1404amd64:~# echo "RM #15259 [HD @52271] Проставляем временные метки" | wc -m
50
```

**Comment by jorisroovers** [comment]
It was quite a bit of work to bring proper unicode support to gitlint across the different supported python versions.

The various required changes were done as part of the following commits: 8d503e01b3ef9bf2e5bf60e4ec083ecc52fdfc55, 0d89f1c02a97d200d0f187395761327041a970c0, d28dba35bc3deea12402d5810f6dac221553984c, 8f6cd8cfcc5346a47565c74cc6042ba1785082a2

Closing this issue now. Fixes will be part of the upcoming 0.8.0 release which is coming very soon (working on the release now).
